tests/pkg_tensorflow-lite/Makefile

```diff
@@ -0,0 +1,21 @@
+# Ensure minimal size by default
+DEVELHELP ?= 0
+
+include ../Makefile.tests_common
+
+# Other available example: hello_world
+EXAMPLE ?= mnist
+
+USEPKG += tensorflow-lite
+
+# internal mnist example is available as an external module
+ifeq (mnist,$(EXAMPLE))
+  DISABLE_MODULE += cortexm_fpu
```

Comment:
Why this?

Reply:
Because without this it crashes on M4/M7 boards. No idea why.

Reply:
Well then add a comment for it?? without one it makes me think it is a requirement. This go for all the other cases in the code.